host/sgx/loadelf.c

```diff
@@ -692,8 +692,17 @@ static oe_result_t _patch(oe_enclave_image_t* image, size_t enclave_end)
     OE_CHECK(
         _set_uint64_t_symbol_value(image, "_reloc_size", image->reloc_size));
 
+#ifdef OE_WITH_EXPERIMENTAL_EEID
+    /* exactly one guard or EEID page */
+    uint64_t eeid_rva = image->image_size + image->reloc_size;
+    OE_CHECK(_set_uint64_t_symbol_value(image, "_eeid_rva", eeid_rva));
+
+    /* heap right after guard/EEID page */
+    oeprops->image_info.heap_rva = eeid_rva + OE_PAGE_SIZE;
+#else
```

Comment:
Once the proposal of adding a guard page between the Code and Initialized Data pages and heap pages in regular enclaves is accepted, the code that adjusts `heap_rva` with OE_PAGE_SIZE should be moved out of the OE_WITH_EXPERIMENTAL_EEID compiling switch.

Reply:
In the updated EEID design proposal, `entry_rva` needs to be included in `oeprops`, for bothe EEID enclaves and regualr enclaves. Once the proposal is adopted, the code to set `entry_rva` probably can be added in this function, and outside of OE_WITH_EXPERIMENTAL_EEID compiling switch.